R/stan_module.R

```diff
@@ -213,12 +187,13 @@ read_file <- function(filename) {
 }
 
 
+#' @importFrom assertthat assert_that
```

Comment:
Hi Craig @gowerc, though the checks passed however I imported random words from the assertthat library, I think the problem is from my moved `#' @importFrom assertthat assert_that` on line 45 before S4 function directly to that before S3 function `is_file`, and it's a matter of roxygen2's syntax. If my guess is right, I'm going to read carefully roxygen2's guide.